pyaedt/generic/touchstone_parser.py

```diff
@@ -475,3 +477,74 @@ def read_touchstone(file_path):
     """
     data = TouchstoneData(touchstone_file=file_path)
     return data
+
+
+@pyaedt_function_handler()
+def check_touchstone_files(folder="", passivity=True, causality=True):
+    """Check passivity and causality for all touchstone files included in folder.
```

Comment:
```suggestion
    """Check passivity and causality for all Touchstone files included in the folder.
```
I did a Google search and see that Touchstone is a proper noun and so should be capitalized in descriptive text.